classes/travel/TravelManager.php

```diff
@@ -332,87 +332,96 @@ public function fetchCurrentMapLocations(): array {
-                    ]),
-                    action_message: "Enter the Fray",
-                );
-            }
-            foreach ($this->system->event->mission_coords['easy'] as $event_mission) {
-                $objectives[] = new MapObjectiveLocation(
-                    name: "Easy",
-                    map_id: 1,
-                    x: $event_mission['x'],
-                    y: $event_mission['y'],
-                    image: "/images/events/lanternred.png",
-                    action_url: $this->system->router->getUrl("mission", [
-                        'start_mission' => $this->system->event->mission_ids['easy_mission_id'], 'mission_type' => 'event'
-                    ]),
-                    action_message: "Begin Search",
-                );
-            }
-            foreach ($this->system->event->mission_coords['medium'] as $event_mission) {
-                $objectives[] = new MapObjectiveLocation(
-                    name: "Medium",
-                    map_id: 1,
-                    x: $event_mission['x'],
-                    y: $event_mission['y'],
-                    image: "/images/events/lanternblue.png",
-                    action_url: $this->system->router->getUrl("mission", [
-                        'start_mission' => $this->system->event->mission_ids['medium_mission_id'], 'mission_type' => 'event'
-                    ]),
-                    action_message: "Follow Signs of battle",
-                );
-            }
-            foreach ($this->system->event->mission_coords['hard'] as $event_mission) {
-                $objectives[] = new MapObjectiveLocation(
-                    name: "Hard",
-                    map_id: 1,
-                    x: $event_mission['x'],
-                    y: $event_mission['y'],
-                    image: "/images/events/lanternviolet.png",
-                    action_url: $this->system->router->getUrl("mission", [
-                        'start_mission' => $this->system->event->mission_ids['hard_mission_id'], 'mission_type' => 'event'
-                    ]),
-                    action_message: "Investigate Suspicious Markings",
-                );
-            }
-            foreach ($this->system->event->mission_coords['nightmare'] as $event_mission) {
-                $objectives[] = new MapObjectiveLocation(
-                    name: "Nightmare",
-                    map_id: 1,
-                    x: $event_mission['x'],
-                    y: $event_mission['y'],
-                    image: "/images/events/yokai_cropped.png",
-                    action_url: $this->system->router->getUrl("mission", [
-                        'start_mission' => $this->system->event->mission_ids['nightmare_mission_id'], 'mission_type' => 'event'
-                    ]),
-                    action_message: "Stop the Ritual",
-                );
+        if (isset($this->system->event)) {
```

Comment:
This shouldn't be necessary, were you seeing an error here?
![image](https://github.com/elementum-games/shinobi-chronicles/assets/1691258/14dd63ad-c765-48c6-8435-51aee554ade9)

Reply:
Testing with/without event active, was throwing an error here I think?